eventuate-core/src/main/scala/com/rbmhtechnology/eventuate/ReplicationEndpoint.scala

```diff
@@ -363,6 +359,96 @@ class ReplicationEndpoint(
 }
 
 /**
+ * An [[EndpointFilter]] computes a [[ReplicationFilter]] that shall be applied to a
+ * replication read request that replicates from a source log (defined by ``sourceLogName``)
+ * to a target log (defined by ``targetLogId``).
+ */
+trait EndpointFilter {
+  def filterFor(targetLogId: String, sourceLogName: String): ReplicationFilter
+}
+
+object EndpointFilter {
```

Comment:
You only talk about the `object` not the `trait`, right? To me `ReplicationFilterCombiner` (or similar) sounds like something that generically combines `ReplicationFilter`s however this is done with a `ReplicationFilter`'s methods (like and, or, ...). `EndpointFilter` is basically a factory for `EndpointFilter`s, so I would prefer `EndpointFilters`. OK?